src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -19,92 +19,89 @@ import aiExplainThisErrorPopUp from 'images/ai_screenshot-crop_explain-this-erro
 
-## Sample questions [#sample-questions] 
+Our AI has been trained to communicate with engineers using common questions and plain language. Let's say you run an online store and start receiving alerts that response times for your `Checkout service` are lagging. If you don't have time to investigate the issue, you can simply ask New Relic AI, "What's on fire?" or "What should I handle first today?" New Relic AI can tell you what's broken in your system and help you explore the data with charts and tables.
 
-If you're struggling with where to begin, here are some prompts. 
+#### Sample questions [#sample-questions] 
+
+If you need help with where to begin, here are some prompts. 
 
 <Tabs>
-    <TabsBar>
-    <TabsBarItem id="1">Get started with New Relic</TabsBarItem>
-    <TabsBarItem id="2">Improve response time</TabsBarItem>
-    <TabsBarItem id="3">Take observability to the next level</TabsBarItem>
-    </TabsBar>
+ <TabsBar>
+ <TabsBarItem id="1">Get started with New Relic</TabsBarItem>
+ <TabsBarItem id="2">Improve response time</TabsBarItem>
+ <TabsBarItem id="3">Take observability to the next level</TabsBarItem>
+ </TabsBar>
 
-    <TabsPages>
+ <TabsPages>
 
-    <TabsPageItem id="1`">
-    Our AI chat experience can help get you started using New Relic. If you need help organizing your data and setting up alerts, NRAI can act as a guide throughout your onboarding process. Ask our AI questions like:
+ <TabsPageItem id="1`">
+ Our AI chat experience can help get you started using New Relic. If you need help organizing your data and setting up alerts, NRAI can act as a guide throughout your onboarding process. Ask our AI questions like:
 
 * How can I use Terraform to create dashboards? 
 * How can I improve my alerts setup?
 * How can I instrument AWS?
-    </TabsPageItem>
+ </TabsPageItem>
 
-    <TabsPageItem id="2">
-    If you want to get to the root cause of latency issues for your app, here are some good questions to ask:
+ <TabsPageItem id="2">
+ If you want to get to the root cause of latency issues for your app, here are some good questions to ask:
 
 * What are some recent incidents in my system?
 * Which infrastructure hosts are showing signs of degradation over the past hour?
-    </TabsPageItem>
+ </TabsPageItem>
 
-    <TabsPageItem id="3">
-    Our AI can also be your partner in observability maturity. If you're responsible for ensuring that your app can handle spikes for a big e-commerce event like Black Friday, it's important to understand the state of your system so you can improve it _ahead of time_. Ask our AI:
+ <TabsPageItem id="3">
+ Our AI can also be your partner in observability maturity. If you're responsible for ensuring that your app can handle spikes for a big e-commerce event like Black Friday, it's essential to understand the state of your system so you can improve it _ahead of time_. Ask our AI:
```

Comment:
While I like the use of Black Friday here as a concrete example, I don't know that it translates well outside of American audiences since we're the only ones who "celebrate" it. Maybe replacing it with something like "a big e-commerce event like launching a new product line," or something like that?

Reply:
Sounds like you both agree! Will make this change!